client/my-sites/site-settings/media-settings-writing.jsx

```diff
@@ -0,0 +1,153 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import JetpackModuleToggle from 'my-sites/site-settings/jetpack-module-toggle';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormSelect from 'components/forms/form-select';
+import FormLabel from 'components/forms/form-label';
+import CompactFormToggle from 'components/forms/form-toggle/compact';
+import SupportInfo from 'components/support-info';
+import isJetpackModuleActive from 'state/selectors/is-jetpack-module-active';
+import isJetpackModuleUnavailableInDevelopmentMode from 'state/selectors/is-jetpack-module-unavailable-in-development-mode';
+import isJetpackSiteInDevelopmentMode from 'state/selectors/is-jetpack-site-in-development-mode';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteSlug } from 'state/sites/selectors';
+import QueryJetpackConnection from 'components/data/query-jetpack-connection';
+import classNames from 'classnames';
```

Comment:
Nit: this is an external import and should be moved up to the "External dependencies" section.